addons/http_routing/models/ir_http.py

```diff
@@ -637,6 +637,12 @@ def _handle_error(cls, exception):
         code, values = cls._get_exception_code_values(exception)
 
         request.cr.rollback()
+        # After changes were rolled back, if the error generates assets bundle
+        # attachments, those must be committed for the error to be properly
+        # displayed.
+        context = dict(request.env.context)
+        context['commit_assetsbundle'] = True
+        request.env = request.env(request.env.cr, request.env.uid, context)
```

Comment:
Please don't reassign the `request.env` attribute yourself, use `request.update_env` or `request.update_context` instead
```suggestion
        request.update_context(commit_assetsbundle=True)
```